src/helpers.cpp

```diff
@@ -596,14 +596,16 @@ bool Helpers::value2bool(const char * value, bool & value_b) {
         return false;
     }
 
-    std::string bool_str = toLower(value); // convert to lower case
+    std::string bool_str = value;
 
-    if ((bool_str == std::string(Helpers::translated_word(FL_(on)))) || (bool_str == "on") || (bool_str == "1") || (bool_str == "true")) {
+    if ((bool_str == std::string(Helpers::translated_word(FL_(on)))) || (bool_str == std::string(Helpers::translated_word(FL_(ON))))
```

Comment:
there are a number of places in Helpers.cpp where we test for boolean and it's translations and alternatives. Maybe good to create a separate function for this like `Helpers::isBool(const char * bool_s)` ?

Reply:
I don't understand what the new function you propose is supposed to do. Can you explain your idea?
In Helpers.cpp (except `value2bool`) I only see `value2enum` functions that test translated boolean names. I missed it. So, `value2enum` should also be rewritten to not use `toLower` helper, but tests both uppercase and lowercase variants. Would you do it?

Reply:
ok, I'll take a look later and refactor myself. it's not that important.

code looks fine, @MichaelDvP ok to merge?